docs/changelog.rst

```diff
@@ -1,6 +1,15 @@
 Changelog
 =========
 
+Version 5.2.0
+-------------
+
+KSQL 5.2 includes new features, including:
+
+* Added a new family of UDFs for improved handling of URIs (e.g. extracting information/decoding information)
```

Comment:
Can you add a link to the docs covering the new functions maybe?  Or the general doc link and the names of the new functions?